nipype/interfaces/fsl/model.py

```diff
@@ -209,9 +209,9 @@ def _create_ev_files(
                             raise Exception('FSL environment variables not set')
                         else:
                             ev_parameters['fsldir'] = '/usr/share/fsl'
-                    try:
-                        ev_parameters['temporalderiv'] = int(bool(ev_parameters.pop('derivs')))
-                    except KeyError:
+                    if 'derivs' in ev_parameters.keys():
```

Comment:
I think this would be more reliable.
```
ev_parameters['temporalderiv'] = bool(ev_parameters.get('derivs', False))
```

BTW, I don't understand why the int casting if `ev_parameters['derivs']` is `True` and then the boolean is left for false.